assets/src/components/nav/navMenu.tsx

```diff
@@ -45,6 +46,10 @@ const NavMenu: React.FC<Props> = ({ mobileMenuIsOpen, toggleMobileMenu }) => {
             <OldCloseIcon className="c-nav-menu__close-icon" />
           </button>
         </div>
+        <div className="c-nav-menu__logged-in-as">
+          <LoggedInAs email="username@mbta.com" />
```

Comment:
Placeholder until https://github.com/mbta/skate/pull/2382 is merged. Then we can use `getEmailAddress()`.